tests/benchmark.py

```diff
@@ -1,15 +1,11 @@
-from concurrent.futures import ThreadPoolExecutor
-from lightning import LightningRpc
-from test_lightningd import NodeFactory
 import logging
-import pytest
 import random
-import utils
-
 from concurrent import futures
 from time import time
+import pytest
+from test_lightningd import NodeFactory
 from tqdm import tqdm
-
+import utils
```

Comment:
Mixing module imports with named imports is wrong IMHO, we should separate them.

Reply:
good catch